### ISSUE TITLE
Adjust the mobile menu for accessibility: 3

### ISSUE THREAD

**Issue by paregorios** [isawnyu/isaw.web#267]
_Epic: #265_

> The mobile menu should have a role = dialogue and have a keyboard trap that keep focus within the modal until the user selects the close button. After the modal is closed, the focus should return back to the mobile menu icon.

WCAG 2.0 Compliance: ???

Traceability: [Microassist Report 1 > WCAG Compliance > Name, Role, Value](https://docs.google.com/document/d/10cfUFx-o_kSGBFKCe4MtL45cyc-c-Gwaz3Dxlrp8Qgg/edit#bookmark=id.t8cmbkue4bcm)

**Comment by paregorios** [comment]
Please merge to master and deploy to production.
